Dockerfile

```diff
@@ -0,0 +1,13 @@
+FROM python:3.14.0a3-slim
+LABEL maintainer="yaroslavbordovoy@gmail.com"
+
+ENV PYTHONUNBUFFERED=1
+
+WORKDIR app/
+
+COPY requirements.txt requirements.txt
+RUN pip install -r requirements.txt
+
+COPY . .
+
+CMD ["python", "app/main.py"]
```

Comment:
Ensure that the path `app/main.py` is correct and that `main.py` is located in the `app` directory. This is crucial for the CMD instruction to work properly.

Reply:
Everything is correct here